src/AppShed/Extensions/StorageBundle/Controller/ApiController.php

```diff
@@ -0,0 +1,444 @@
+<?php
+/**
+ * Created by mcfedr on 05/05/2014 21:31
+ */
+
+namespace AppShed\Extensions\StorageBundle\Controller;
+
+use AppShed\Extensions\StorageBundle\Entity\Api;
+use AppShed\Extensions\StorageBundle\Entity\Data;
+use AppShed\Extensions\StorageBundle\Entity\Field;
+use AppShed\Extensions\StorageBundle\Entity\Filter;
+use AppShed\Extensions\StorageBundle\Form\ApiEditType;
+use AppShed\Extensions\StorageBundle\Form\ApiType;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+class ApiController extends StorageController
+{
+    /**
+     * @Route("/api/list", name="api_list")
+     * @Method({"GET"})
+     * @Template()
+     */
+    public function listAction(Request $request)
+    {
+        $app = $this->getApp($request);
+        $data['apis'] = $this->getDoctrine()->getRepository('AppShedExtensionsStorageBundle:Api')->findBy(['app' => $app]);
+        $data['appParams'] = $this->getExtensionAppParameters($request);
+        return $data;
```

Comment:
I would rather you wrote 

    return [
        'apis' => ...,
        'appParams' => ....
    ];